lib/auth/auth_with_roles.go

```diff
@@ -2821,9 +2821,6 @@ func (a *ServerWithRoles) UpsertSAMLConnector(ctx context.Context, connector typ
 	if err := a.authConnectorAction(apidefaults.Namespace, types.KindSAML, types.VerbUpdate); err != nil {
 		return trace.Wrap(err)
 	}
-	if !modules.GetModules().Features().SAML {
```

Comment:
Wonder if there is a case when a client can have a Enterprise product without SAML feature ?

Reply:
Not currently as far as I'm aware. This check is still performed but in a different way. If the enterprise startup does not register the SAML connector, you'll still get this error message from the code in `lib/auth/saml.go` in the OSS repo. If we do get a scenario where we can have an enterprise product without SAML, we can revise the error message then since I don't know how to word it otherwise right now.